### PR TITLE
Undertaker model fix & Medieval textures

### PR DESCRIPTION
**Pull request by carlansor**
Fixes wrong head rendering
Added medieval undertaker textures

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/client/model/ModelEntityUndertakerMale.java [hunk 1]
```diff
@@ -69,6 +69,8 @@ public ModelEntityUndertakerMale()
         bipedLeftLeg.setRotationPoint(1.9F, 12.0F, 0.0F);
         bipedLeftLeg.setTextureOffset(16, 48).addBox(-2.0F, 0.0F, -2.0F, 4.0F, 12.0F, 4.0F, 0.0F, false);
         bipedLeftLeg.setTextureOffset(0, 48).addBox(-2.0F, 0.0F, -2.0F, 4.0F, 12.0F, 4.0F, 0.25F, false);
+
```
Comment (ravenbuilder934) [inline src/main/java/com/minecolonies/coremod/client/model/ModelEntityUndertakerMale.java:72]: inconsistent whitespace when compared with female model :angery:

**Review by ravenbuilder934 — DISMISSED** [review]

File: src/main/java/com/minecolonies/coremod/client/model/ModelEntityUndertakerMale.java [hunk 2]
```diff
@@ -69,6 +69,8 @@ public ModelEntityUndertakerMale()
         bipedLeftLeg.setRotationPoint(1.9F, 12.0F, 0.0F);
         bipedLeftLeg.setTextureOffset(16, 48).addBox(-2.0F, 0.0F, -2.0F, 4.0F, 12.0F, 4.0F, 0.0F, false);
         bipedLeftLeg.setTextureOffset(0, 48).addBox(-2.0F, 0.0F, -2.0F, 4.0F, 12.0F, 4.0F, 0.25F, false);
+		
+	bipedHeadwear.showModel = false;
```
Comment (ToMe25) [inline src/main/java/com/minecolonies/coremod/client/model/ModelEntityUndertakerMale.java]:
```suggestion
        bipedHeadwear.showModel = false;
```
The formatting here doesn't match either.

File: src/main/java/com/minecolonies/coremod/client/model/ModelEntityUndertakerFemale.java [hunk 3]
```diff
@@ -78,6 +78,7 @@ public ModelEntityUndertakerFemale()
         bipedLeftLeg.setRotationPoint(1.9F, 12.0F, 0.0F);
         bipedLeftLeg.setTextureOffset(16, 48).addBox(-2.0F, 0.0F, -2.0F, 4.0F, 12.0F, 4.0F, 0.0F, true);
         bipedLeftLeg.setTextureOffset(0, 48).addBox(-2.0F, 0.0F, -2.0F, 4.0F, 12.0F, 4.0F, 0.25F, false);
+	bipedHeadwear.showModel = false;
```
Comment (ToMe25) [inline src/main/java/com/minecolonies/coremod/client/model/ModelEntityUndertakerFemale.java]:
```suggestion
        bipedHeadwear.showModel = false;
```
The formatting here doesn't match :stuck_out_tongue:

**Review by ToMe25 — COMMENTED** [review]
I saw ravenbuilders review and thought I'd look over this, and found some formatting nitpicks.

**Pull request merged**